tests/helpers/userHelper.py

```diff
@@ -0,0 +1,73 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+
+
+def userRegister(details,driver):
+    element = driver.find_element_by_xpath("//div[@class='mat-typography']/div/button[@class='mat-focus-indicator close-dialog mat-raised-button mat-button-base mat-primary ng-star-inserted']").click()
+    element = driver.find_element_by_xpath("//div[@class='cc-window cc-floating cc-type-info cc-theme-classic cc-bottom cc-right cc-color-override-1934802758 ']/div[@class='cc-compliance']").click()
+
+
+    element = driver.find_element_by_xpath("//*[@class='mat-toolbar-row']/button[@class='mat-focus-indicator mat-menu-trigger buttons mat-button mat-button-base']").click()
+    element = driver.find_element_by_id("navbarLoginButton").click()
+    
+    #driver.implicitly_wait(10)
+    wait = WebDriverWait(driver, 10)
+    wait.until(ec.presence_of_element_located((By.ID, "newCustomerLink")))
+    element = driver.find_element_by_id('newCustomerLink')
+    element.click()
+
+    #driver.implicitly_wait(10)
+    wait.until(ec.presence_of_element_located((By.ID,'emailControl')))
+    email = driver.find_element_by_id('emailControl')
```

Comment:
Please create a function that fills inputs so that you can add code readability and simplicity